README.md

```diff
@@ -24,6 +24,11 @@ To allow for ease of deployment the code is kept in separate folders, chrome, an
 
 # Version history
 
+### 2.1.1 
```

Comment:
could this actually be 2.1.2? because I had to publish 2.1.1 to disable the ajax calls